dht_test.go

```diff
@@ -1275,6 +1275,10 @@ func TestFindClosestPeers(t *testing.T) {
 		connect(t, ctx, dhts[i], dhts[(i+1)%len(dhts)])
 	}
 
+	for i := 0; i < nDHTs; i++ {
+		dhts[i].BootstrapOnce(ctx, DefaultBootstrapConfig)
+	}
+
```

Comment:
So, I spent a while trying to avoid having to fix this test without complicating the query logic too much. Then I realized that was just stupid.